connectors/v2/pakartot.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#player';
+
+Connector.artistSelector = 'div.jp-player-artist > a';
+
+Connector.trackSelector = 'div.jp-player-title > a';
+
+Connector.playButtonSelector = '.jp-play';
+
+Connector.getArtist = function() {
+	var text = $(this.artistSelector).text().trim();
```

Comment:
Looks nice for a start. But a lot of cleanup functions will cause 'flags hell'. I have some idea about configurable cleanups without flags in `connector` module (and it can be combined with Youtube shared metadata filter!), but I'm afraid it's will be too complicated. And I'm not sure if it is implementable since I'm not familiar with JavaScript world. I'll try to implement it laler, and we'll see how it's can be used.

P.S. I noticed we all forgot about Gitter and started to discuss things in diff comments :D